SampleGetApi/Tests/FreeApiTest.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Newtonsoft.Json.Linq;
using RestSharp;
using AventStack.ExtentReports;
using Xunit.Abstractions;
using SampleGetApi.Tests;
using AventStack.ExtentReports.MarkupUtils;
+
+namespace SampleGetApi
{
    // Collection attribute to group related tests
    [Collection("SampleApi")]
    public class SampleGetApi : BaseClass
    {
+        // Public reference to RestClient
+        public RestClient restClient;
+
+        // Constructor to initialize RestClient at the start of the scripts
+        public SampleGetApi(ITestOutputHelper output) : base(output)
        {
            // Creating a reference to RestClient and setting the Base Url
            restClient = new RestClient(new RestClientOptions
            {
                BaseUrl = new Uri("https://catfact.ninja/")
            });
+            // Logging test status information
+            _test?.Log(Status.Info, "RestClient initialized with base URL: https://catfact.ninja/");
        }

        [Theory]
        [InlineData("1")]
        [InlineData("2")]
        [InlineData("-3")]
        public void GetCatBreeds(string limitNumber)
        {
            // Creating a RestRequest for getting cat breeds with a limit
            var breedRequest = new RestRequest("/breeds");
            _test?.Log(Status.Info, "Request path is : " + "/breeds");
```

Comment:
Not sure why I see some special characters

Reply:
Not sure will raise one more MR with all the mentioned changes